contracts/libs/MarginAccount.sol

```diff
@@ -1,12 +1,220 @@
-/**
- * SPDX-License-Identifier: UNLICENSED
- */
-pragma solidity 0.6.10;
+pragma solidity =0.6.10;
+
+import {SafeMath} from "../packages/oz/SafeMath.sol";
 
 /**
- *
+ * SPDX-License-Identifier: UNLICENSED
+ * @dev The MarginAccount is a library that provides Controller with an Account of Vault structs, and
+ * the functions that manipulate vaults. Vaults describe positions that users have.
  */
-// solhint-disable-next-line no-empty-blocks
 library MarginAccount {
+    using SafeMath for uint256;
+
+    /**
+     * @dev Account is a struct corresponding to a user that describes how many vaults that user has.
```

Comment:
@haythem96 @antoncoding what do we want to use for consistency? @dev or @notice?